packages/bundle-source/src/tool.js

```diff
@@ -196,7 +197,7 @@ export const makeBundleCache = (wr, cwd, readPowers, opts) => {
 };
 
 export const main = async (args, { fs, url, crypto, path }) => {
-  const [to, dest, ...pairs] = args;
+  const [to, dest, ...pairs] = /** @type {[string, string, string[]]} */ (args);
```

Comment:
please type the function signature than cast. fine with any on the second param: `{*} io`